File: lpg2/tests/golden/minimal/csharp/minimalsym.cs
public interface minimalsym {
    public const int
      a = 1,
      EOF_TOKEN = 2;

    public static string[] orderedTerminalSymbols = {
                 "",
                 "a",
                 "EOF_TOKEN"
             };

    public const int numTokenKinds = 2;


    public const bool isValidForParser = true;
}

File: lpg2/tests/golden/minimal/python3/minimalsym.py
class minimalsym(object):
   a: int  = 1
   EOF_TOKEN: int  = 2

   orderedTerminalSymbols: list = [
                 "",
                 "a",
                 "EOF_TOKEN"
             ]

   numTokenKinds: int = 2

   isValidForParser : bool = True

